ale_linters/perl/perlcritic.vim

```diff
@@ -1,17 +1,32 @@
-" Author: Vincent Lequertier <https://github.com/SkySymbol>
+" Author: Vincent Lequertier <https://github.com/SkySymbol>, Chris Weyl <cweyl@alumni.drew.edu>
 " Description: This file adds support for checking perl with perl critic
 
-if !exists('g:ale_perl_perlcritic_showrules')
-    let g:ale_perl_perlcritic_showrules = 0
-endif
+let g:ale_perl_perlcritic_executable =
+\   get(g:, 'ale_perl_perlcritic_executable', 'perlcritic')
+
+let g:ale_perl_perlcritic_profile =
+\   get(g:, 'ale_perl_perlcritic_profile', '.../.perlcriticrc')
+
+let g:ale_perl_perlcritic_options =
+\   get(g:, 'ale_perl_perlcritic_options', '')
+
+let g:ale_perl_perlcritic_showrules =
+\   get(g:, 'ale_perl_perlcritic_showrules', 0)
+
+function! ale_linters#perl#perlcritic#GetExecutable(buffer) abort
+    return ale#Var(a:buffer, 'perl_perlcritic_executable')
+endfunction
 
 function! ale_linters#perl#perlcritic#GetCommand(buffer) abort
     let l:critic_verbosity = '%l:%c %m\n'
-    if g:ale_perl_perlcritic_showrules
+    if ale#Var(a:buffer, 'perl_perlcritic_showrules')
         let l:critic_verbosity = '%l:%c %m [%p]\n'
     endif
 
-    return "perlcritic --verbose '". l:critic_verbosity . "' --nocolor"
+    return ale_linters#perl#perlcritic#GetExecutable(a:buffer)
```

Comment:
You should apply `ale#Escape` to the executable here if it's configurable, as the executable path could contain spaces, etc.